src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt

```diff
@@ -79,17 +118,17 @@ class ExternalDocRequestEndpointIntTest : IntegrationTestBase() {
         verify(telemetryService).trackEvent(
             TelemetryEventType.COURT_LIST_MESSAGE_RECEIVED,
             mapOf(
-                "sqsMessageId" to "a4e9ab53-f8aa-bf2c-7291-d0293a8b0d02",
+
                 "courtCode" to "B10JQ",
                 "courtRoom" to "0",
                 "hearingDate" to "2020-10-26",
                 "fileName" to "5_26102020_2992_B10JQ00_ADULT_COURT_LIST_DAILY"
             )
         )
-        verify(sqsService).enqueueMessage(anyString())
-        val expectedMessageDetail = MessageDetail(courtCode = "B10JQ", courtRoom = 0, hearingDate = "2020-10-26")
-        verify(s3Service).uploadMessage(eq(expectedMessageDetail), contains("ExternalDocumentRequest"))
-        verifyNoMoreInteractions(sqsService, s3Service)
+
+        checkMessage(amazonSQS.receiveMessage(ReceiveMessageRequest(queueUrl)).messages[0], 166662981)
```

Comment:
Can we create a static var for the case number, rather than a magic number in the code.

Reply:
We are now checking some more case details to make the test clearer